charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -64,6 +64,7 @@ spec:
             {{- if .Values.controller.nodeFinalize.skipped }}
             - --skip-node-finalize
             {{- end }}
+            - --profiling-port={{ .Values.controller.profiling.port | default 0 }}
```

Comment:
We should update this. Must check other places.

Reply:
:man_facepalming: I didn't test this after the last update or I would have caught this. I'll get this fixed. Thanks.

Reply:
Ok, these have all been updated, and tested with the helm chart in minikube.